website/content/guides/1.7-template-upgrade.mdx

```diff
@@ -8,50 +8,71 @@ sidebar_title: Upgrade Your Template to use Packer init
 The `packer init` command introduced in version v1.7.0 benefits Packer users by providing better control of plugin
 usage within a Packer template configuration.
 
-In short words, `packer init` will look at the `required_plugins` block definition within a template, for the required
-plugins specifications, and download the correct binaries for that template. For details about how the command works and
-where the plugins will be installed, please refer to the [`packer init`](/docs/commands/init) documentation.
+In short, `packer init` will look at the `required_plugins` block definition within a template, and download the correct
+binaries for that template. For details about how the command works and where the plugins will be installed, please
+refer to the [`packer init`](/docs/commands/init) documentation.
 
 -> **Note:** `packer init` is only supported for HCL templates. You can
 upgrade legacy JSON templates to HCL using the [hcl2_upgrade](/docs/commands/hcl2_upgrade) command.
 
 ## Updating your template with the `required_plugins` block
 
-### Why do I need to upgrade my template?
+## FAQs
 
-To use the `packer init` command and have all the winnings that it brings, you must upgrade
-your template to use the `required_plugins` block. The `init` command won't work if no `required_plugins` is provided.
+### Why do you need to upgrade your template?
```

Comment:
I changed "my" to "your" in all these titles to make it more consistent across the doc.